lib/patch_ruby/api/orders_api.rb

```diff
@@ -160,23 +161,22 @@ def create_order_with_http_info(create_order_request, opts = {})
     end
 
     # Place an order
-    # Placing an order confirms an order's allocation of offsets. Only orders that are in the `draft` state can be placed
-    # @param id [String]
+    # Placing an order confirms an order's allocation of offsets. Only orders that are in the `draft` state can be placed 
+    # @param id [String] 
     # @param [Hash] opts the optional parameters
-    # @option opts [PlaceOrderRequest] :place_order_request
+    # @option opts [PlaceOrderRequest] :place_order_request 
     # @return [OrderResponse]
-    def place_order(order_id, opts = {})
-      place_order_request = Patch::PlaceOrderRequest.new(opts)
-      opts[:place_order_request] = place_order_request
-      data, _status_code, _headers = place_order_with_http_info(order_id, opts)
+    def place_order(id, opts = {})
+    
+      data, _status_code, _headers = place_order_with_http_info(id, opts)
       data
     end
```

Comment:
This version had (most likely) be written manually. The new one is entirely generated by the templates.